modules/rosa-sts-creating-cluster.adoc

```diff
@@ -172,12 +189,31 @@ $ for role in `find . -name "*-role.json"`
 do 
   policy=$(sed -e 's/05-/06-/' -e 's/role/policy/' <<< ${role})
   role_name=$(grep RoleName ${policy} | awk '{print $2}' | awk -F '"' '{print $2}')
-  aws iam create-role --cli-input-json file://${role} [--permissions-boundary ${permissions_boundary_arn}]
+  aws iam create-role --cli-input-json file://${role}
   sed -i.bak '/RoleName/d' ${policy}
   policy_arn=$(aws iam create-policy --cli-input-json file://$policy | grep Arn | awk '{print $2}' | awk -F '"' '{print $2}')
   aws iam attach-role-policy --role-name $role_name --policy-arn $policy_arn
   rm ${policy}
   mv ${policy}.bak ${policy}
   sleep 5 # Prevents AWS Rate limiting
 done
-----
\ No newline at end of file
+----
++
+[NOTE]
+====
+If relying on a permissions boundary arn, use the following `aws iam create-role` command in the above loop
+[source,terminal]
+----
+$ aws iam create-role \
+    --cli-input-json file://${role} \
+    --permissions-boundary ${permissions_boundary_arn}
+----
+====
++
+. Enter the following command to follow the OpenShift installer logs to track the progress of your cluster creation:
```

Comment:
Consider this edit: To track the progress of your cluster creation, enter this command to watch the OpenShift installer logs.